README.md

```diff
@@ -49,16 +49,18 @@ The following build events are included in the project file:
 - Post-Build event (`demo`): `copy $(ProjectDir)lib\play_cpp_sdk.dll $(TargetDir)`
 
 ### Mac
-1. Clone the current repository
+1. Clone the current repository 
     ``` sh
     git clone https://github.com/crypto-com/play-cpp-sdk.git
     ```
 2. Unzip the archive file into `demo` folder
-3. Copy the dynamic library to `/usr/local/lib`
+3. Copy the dynamic library to your dylib folder, for example $HOME/lib
     ``` sh
     cd demo
-    cp lib/libplay_cpp_sdk.dylib /usr/local/lib
+    cp lib/libplay_cpp_sdk.dylib $HOME/lib
```

Comment:
$HOME/lib maybe make things complicated... See https://stackoverflow.com/questions/57318463/macos-whats-the-correct-place-to-install-a-dylib-on-a-users-system#:~:text=%2FLibrary%2FApplication%20Support%20or%20%2FLibrary%2FFrameworks

Reply:
Of course, it is ok to add a new $HOME/lib and export it with `DYLD_LIBRARY_PATH`. It is still a workaround... Ideally, most users just simply run the program in terminal (like other programs) without touching the env.

Reply:
ok, i'll remove it